saleor/payment/gateways/stripe/webhooks.py

```diff
@@ -45,17 +52,20 @@ def handle_webhook(request: WSGIRequest, gateway_config: "GatewayConfig"):
         return HttpResponse(status=400)
 
     # Handle the event
-    if event.type == "payment_intent.succeeded":
-        payment_intent = event.data.object  # contains a stripe.PaymentIntent
+    if event.type == WEBHOOK_SUCCESS_EVENT:
+        payment_intent = event.data.object
         handle_successful_payment_intent(payment_intent, gateway_config)
-    # TODO handle rest of the events will be added in separate PR
-    # elif event.type == 'payment_method.attached':
-    #     payment_method = event.data.object  # contains a stripe.PaymentMethod
-    #     print('PaymentMethod was attached to a Customer!')
-    # # ... handle other event types
-    # else:
-    #     print('Unhandled event type {}'.format(event.type))
-
+    elif event.type == WEBHOOK_AUTHORIZED_EVENT:
+        payment_intent = event.data.object
+        handle_authorized_payment_intent(payment_intent, gateway_config)
+    elif event.type == WEBHOOK_PROCESSING_EVENT:
+        payment_intent = event.data.object
+        handle_processing_payment_intent(payment_intent, gateway_config)
+    elif event.type in [WEBHOOK_FAILED_EVENT, WEBHOOK_CANCELED_EVENT]:
+        payment_intent = event.data.object
+        handle_failed_payment_intent(payment_intent, gateway_config)
+    else:
+        logger.warning("Received unhandled webhook event %s", event.type)
     return HttpResponse(status=200)
```

Comment:
Every condition here is similar so it can be simplified with the use of mapping like:
```suggestion
webhook_type_to_handler_mapping = {
    WEBHOOK_SUCCESS_EVENT: handle_successful_payment_intent,
    WEBHOOK_AUTHORIZED_EVENT: handle_authorized_payment_intent,
    ...
}
if event.type in webhook_type_to_handler_mapping:
    payment_intent = event.data.object
    webhook_type_to_handler_mapping[event.type](payment_intent, gateway_config)
else:
    logger.warning("Received unhandled webhook event %s", event.type)
```
For me, it's more readable, as I don't need to check every condition separately.